src/entry.js

```diff
@@ -29,7 +29,18 @@ function StacheEntryPlugin() {
     return content;
   };
 
-  return Object.freeze({ preload });
+  const runCommand = (command) => {
+    switch(command) {
+      case 'stache-update':
+        require('./cli-commands/update-dependencies')();
+      break;
+      default:
+      return false;
+    }
+    return true;
+  };
+
+  return Object.freeze({ preload, runCommand });
 }
 
 module.exports = StacheEntryPlugin;
```

Comment:
I would advise keeping these two concerns separate. For the main index.ts file, perhaps you could export what's needed, but keep the plugin stuff separate from the CLI commands?

**Untested example:**

```
const cliCommands = require('./src/cli-commands');
const StacheEntryPlugin = require('./src/entry');
const plugin = new StacheEntryPlugin();

const publicApi = {};
publicApi.preload = plugin.preload;
publicApi.runCommand = cliCommands.runCommand,

module.exports = publicApi;
```

Reply:
Makes sense to me!